daemon/api_snaps.go

```diff
@@ -477,24 +475,29 @@ func snapInstall(ctx context.Context, inst *snapInstruction, st *state.State) (*
 	}
 
 	return &snapInstructionResult{
-		Summary:            installMessage(inst, ckey),
+		Summary:            installRefreshMessage(inst.Snaps[0], inst.CompsForSnaps[inst.Snaps[0]], inst.Action, inst.Channel, inst.CohortKey),
 		Tasksets:           tss,
 		Affected:           installedSnaps,
 		AffectedComponents: installedComponents,
 	}, nil
 }
 
-func installMessage(inst *snapInstruction, cohort string) string {
+func installRefreshMessage(snap string, comps []string, action, channel, cohort string) string {
```

Comment:
instead of passing all things should have we not just simply added passing a snap name in?

Reply:
Yeah you're right, that does look better. Changed now.